package.json

```diff
@@ -1150,6 +1186,26 @@
           "default": false,
           "description": "Remove hidden items when clearing workspace."
         },
+        "r.liveShare.timeout": {
+          "type": "integer",
+          "default": 3000,
+          "description": "Time in milliseconds before aborting attempt to connect to Live Share API"
+        },
+        "r.liveShare.commandForwardDefault": {
```

Comment:
I guess we could follow the naming style `r.liveShare.defaults.commandForward`, `r.liveShare.defaults.shareWorkspace`, `r.liveShare.defaults.shareBrowser`?

It is consistent with the defaults introduces in https://github.com/Ikuyadeu/vscode-R/pull/620/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R1153.